app/src/main/java/org/fossasia/openevent/general/social/SocialLinksViewModel.kt

```diff
@@ -1,5 +1,6 @@
 package org.fossasia.openevent.general.social
 
+import android.content.Context
```

Comment:
There should be no imports of android.* in ViewModel

Reply:
how can I use context then?

Reply:
You can't. That's the point of ViewModel. Each time a programmer uses context or view reference in ViewModel, MVVM dies a thousand deaths and rolls in its grave

Reply:
lol, so what's the solution?

Reply:
See how strings are getting loaded in the ViewModel

Reply:
through injection?

Reply:
Did it just like the resource